bin/_mocha

```diff
@@ -231,6 +231,7 @@ program
   .option('--trace-deprecation', 'show stack traces on deprecations')
   .option('--trace-warnings', 'show stack traces on node process warnings')
   .option('--use_strict', 'enforce strict mode')
+  .option('--allow-natives-syntax', 'allow natives syntax')
```

Comment:
Well, it's at least displayed. Still have no idea what it does though...

Reply:
Actually, i just documented that flag. Maybe this PR [Pass `--allow-natives-syntax` flag to node.](https://github.com/mochajs/mocha/pull/1621) will give some context about that flag.

Reply:
Maybe just a hint then:
```js
  .option('--allow-natives-syntax', 'allow direct calls to V8 native commands')
```

Reply:
Great. The hint you added is more clearly.